cmd/manager/main.go

```diff
@@ -76,6 +68,22 @@ func main() {
 
 	pflag.Parse()
 
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
+			os.Exit(0)
+		default:
+			fmt.Printf("unknown sub-command: %v\n", os.Args[1])
+			fmt.Print("available sub-commands:\n\tversion\n")
+			fmt.Print("available flags:\n")
+			pflag.PrintDefaults()
```

Comment:
Does this actually provide any useful information given we don't support any flags?

Reply:
WMCO doesn't have its own flags, but we have flags for `zap` and `kubeconfig` which I thought will be helpful if printed.

Reply:
Then please add a comment saying that is what is being done otherwise it is confusing.